src/cli/self_update.rs

```diff
@@ -166,7 +166,7 @@ To get started you need Cargo's bin directory ({cargo_home}/bin) in your `PATH`
 environment variable. Next time you log in this will be done
 automatically.
 
-To configure your current shell run `source {cargo_home}/env`
+To configure your current shell run :\n`source {cargo_home}/env`\n
```

Comment:
Rather than embedded `\n`'s here, just insert the extra lines you wish to show, as this is a multi-line string already.